.travis.yml

```diff
@@ -1,6 +1,11 @@
 ---
 language: ruby
 
+addons:
+  apt:
+    packages: 
+      - lsb-release
```

Comment:
Thanks for spotting this. I think the LSB facts are a vestige of having to support Suse 9, which did not support the non-LSB facts. Not sure if that's still the case. Hopefully something we can refactor when this moves to Puppet v4 only.